### PR TITLE
Store form/sections/services in Redux, read form data on Form.jsx

### PR DESCRIPTION
**Pull request by kebrahim**
Works on #12 #13 

In App.js, all API data is stored in Redux, separated by category. In Form.jsx, form data is read from Redux & keys are displayed on the page, as a proof-of-concept:

qkuzH2wVJ8s

### CODE REVIEW

File: client/src/app/store.js [hunk 1]
```diff
@@ -2,11 +2,17 @@ import {configureStore} from '@reduxjs/toolkit';
...
+import formReducer from '../features/form/formSlice';
+import sectionsReducer from '../features/sections/sectionsSlice';
+import servicesReducer from '../features/services/servicesSlice';
 
 export default configureStore({
   reducer: {
     counter: counterReducer,
     questions: questionsReducer,
     responses: responsesReducer,
+    form: formReducer,
```
Comment (marcja) [inline client/src/app/store.js:14]: Nit: the order of these reducers seems random, neither alphabetical nor semantic (form/section/questions/responses/servers).

File: client/src/components/Form.jsx [hunk 2]
```diff
@@ -1,38 +1,45 @@
...
-    return (
-      <ul>
-        {renderQuestions(questionKeys)}
-      </ul>
-    );
+    const renderQuestionKeys =
+        (questionKeys) => questionKeys.map(renderQuestionKey);
+    const renderSection = (section) => {
+      const sectionKey = Object.keys(section)[0];
+      const questionKeys = Object.values(section)[0];
```
Comment (marcja) [inline client/src/components/Form.jsx:26]: Or:

```javascript
const [sectionKey, questionKeys] = Object.entries(section)[0];
```

File: client/src/app/store.js (hunk 1 above)
Comment (kebrahim) [reply client/src/app/store.js]: done

File: client/src/components/Form.jsx (hunk 2 above)
Comment (kebrahim) [reply client/src/components/Form.jsx]: done

**Pull request merged**